pkgs/by-name/kl/klocalizer/package.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3Packages
+}:
+
+let
+  version = "4.5.1";
```

Comment:
Personal nit: I like to let-block pname, version and src.
It is useful because changelog usually points to src.rev.